mara_pipelines/commands/files.py

```diff
@@ -176,3 +178,57 @@ def html_doc_items(self) -> [(str, str)]:
                  _.tt[json.dumps(self.null_value_string) if self.null_value_string != None else None]),
                 ('time zone', _.tt[self.timezone]),
                 (_.i['shell command'], html.highlight_syntax(self.shell_command(), 'bash'))]
+
+
+class WriteFile(sql._SQLCommand):
+    """Writes data from a local file"""
```

Comment:
This should mention that it's local to the runner, so in some (most nowadays?) cases not useful (e.g. when running it in a ecs fargate task or on a k8s cluster)

Reply:
Right. It assumes that you mount remote storages to a local path e.g. `/mnt/my_datalake`.
I have a newer version for `WriteFile` which uses package [mara-storage](https://github.com/mara/mara-storage). But since version 3.x does not yet implement mara_storage (see #80), I removed that part. Will add another PR for this after this one is merged.

p.s. if you're still interested: mara-storage would need an implementation for AWS S3. I would love to see someone contributing a PR for this.

Reply:
@jankatins I now relalized that I did not get your point. It is true, it is executed locally which is not nowadays a thing to do. To fix this, I think https://github.com/mara/mara-pipelines/issues/68 is the right option. Support for other storages is planned as mentioned above.